AEPIdentity/Sources/IdentityState.swift

```diff
@@ -57,12 +84,17 @@ class IdentityState {
         // Update hit queue with privacy status
         hitQueue.handlePrivacyChange(status: identityProperties.privacyStatus)
 
-        hasBooted = true
-        Log.debug(label: "\(LOG_TAG):\(#function)", "Identity has successfully booted up")
         // Identity should always share its state
         // However, don't create a shared state twice, which will log an error
-        // The force sync event processed above will create a shared state if the privacy is not opt-out
-        return syncIdentifiers(event: event) != nil || identityProperties.privacyStatus == .optedOut
+        hasSynced = syncIdentifiers(event: event) != nil || identityProperties.privacyStatus == .optedOut
+
+        // If the sync was susccessful and there is no shared state update, post a shared state update
+        if hasSynced && !hasBooted {
```

Comment:
See my comment above, this would become: 
```
if hasSynced && !didCreateFirstSharedState {
     createSharedState(identityProperties.toEventData(), event)
     didCreateBootSharedState = true
}